esrally/resources/annotation-template.json

```diff
@@ -0,0 +1,47 @@
+{
+  "settings": {
+    "number_of_shards": 1
+  },
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "strings": {
+          "match": "*",
+          "match_mapping_type": "string",
+          "mapping": {
+            "type": "keyword"
+          }
+        }
+      }
+    ],
+    "properties": {
+      "environment": {
+        "type": "keyword"
+      },
+      "race-timestamp": {
+        "type": "date",
+        "fields": {
+          "raw": {
+            "type": "keyword"
+          }
+        },
+        "format": "basic_date_time_no_millis"
+      },
+      "user-tag": {
+        "type": "keyword"
+      },
+      "track": {
+        "type": "keyword"
+      },
+      "chart": {
+        "type": "keyword"
+      },
+      "chart-name": {
+        "type": "keyword"
+      },
+      "message": {
+        "type": "keyword"
```

Comment:
I know we currently (night-rally) maps the message field as a keyword, but perhaps we should consider using text?

Reply:
would would be the benefits?

Reply:
Probably a little contrived, but it's the only field we'd actually want to perform full-text search on (rather than group by/terms queries etc.). It's not a huge deal, but I thought it might be worth doing upfront rather than reindexing later. I'll leave it up to you :-)